### PR TITLE
feat: Curate explorer listing to only show featured projects

### PR DESCRIPTION
**Pull request by Elyniss**

### CODE REVIEW

File: core/android/src/main/kotlin/com/walletconnect/android/internal/common/explorer/data/network/ExplorerService.kt [hunk 1]
```diff
@@ -20,7 +20,8 @@ interface ExplorerService {
         @Query("projectId") projectId: String,
         @Query("entries") entries: Int,
         @Query("page") page: Int,
-        @Query("is_verified") isVerified: Boolean,
+        @Query("isVerified") isVerified: Boolean,
+        @Query("isFeatured") isFeatured: Boolean,
```
Comment (TalhaAli00) [inline core/android/src/main/kotlin/com/walletconnect/android/internal/common/explorer/data/network/ExplorerService.kt:24]: Were these updated on purpose from using underscores to camel case?

File: core/android/src/main/kotlin/com/walletconnect/android/internal/common/explorer/domain/usecase/GetProjectsWithPaginationUseCase.kt [hunk 2]
```diff
@@ -6,6 +6,6 @@ import com.walletconnect.android.internal.common.explorer.data.model.Project
 class GetProjectsWithPaginationUseCase(
     private val explorerRepository: ExplorerRepository,
 ) {
-    suspend operator fun invoke(page: Int, entries: Int, isVerified: Boolean): Result<List<Project>> =
-        runCatching { explorerRepository.getProjects(page, entries, isVerified).projects }
+    suspend operator fun invoke(page: Int, entries: Int, isVerified: Boolean, isFeatured: Boolean): Result<List<Project>> =
+        runCatching { explorerRepository.getProjects(page, entries, isVerified, isFeatured).projects.sortedBy { it.order ?: Long.MAX_VALUE } }
```
Comment (TalhaAli00) [inline core/android/src/main/kotlin/com/walletconnect/android/internal/common/explorer/domain/usecase/GetProjectsWithPaginationUseCase.kt:10]: With the fallback to max value, items that don't have an order will be moved to the top. Is that the desired effect? I would imagine they get moved down

**Review by TalhaAli00 — APPROVED** [review]

**Review by jakubuid — APPROVED** [review]

File: core/android/src/main/kotlin/com/walletconnect/android/internal/common/explorer/domain/usecase/GetProjectsWithPaginationUseCase.kt (hunk 2 above)
Comment (kacperoak) [reply core/android/src/main/kotlin/com/walletconnect/android/internal/common/explorer/domain/usecase/GetProjectsWithPaginationUseCase.kt:10]: I think this fallback is ok. Max value will put them at the end of the list, which seems correct for me

**Review by kacperoak — APPROVED** [review]

File: core/android/src/main/kotlin/com/walletconnect/android/internal/common/explorer/data/network/ExplorerService.kt (hunk 1 above)
Comment (Elyniss) [reply core/android/src/main/kotlin/com/walletconnect/android/internal/common/explorer/data/network/ExplorerService.kt:24]: yes, saw it on browser networking calls, and decided to change to match them

**Pull request merged**